frontend/src/components/layout/AuthContainer.vue

```diff
@@ -11,7 +11,7 @@
       </v-card>
     </v-col>
     <span v-if="$vuetify.breakpoint.smAndUp" class="photo_credits ma-3 pa-1">
-      Photo by Markus Rohner / Lotos
+      {{ $tc('views.auth.container.photoCredits') }}
```

Comment:
Key should be `components.layout.authContainer.photoCredits` (also remember to change it in the JSON)